gc-google-sheets/gcgs-custom-where-clause.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Gravity Connect // Google Sheets // Custom Where Clause
+ *
+ * This snippet customizes the generation of `where` clauses for Google Sheets queries.
+ *
+ * Installation:
+ *   1. Install per https://gravitywiz.com/documentation/how-do-i-install-a-snippet/
+ */
+add_filter( 'gcgs_gppa_build_gviz_where_clause', function( $clause, $clauses, $value, $column_letter, $operator, $args ) {
+	$conditions = array();
+
```

Comment:
Pending [the filter addition](https://github.com/gravitywiz/gc-google-sheets/pull/163)

I noticed that this sometimes errored when `$value` is an empty string

```suggestion
if ( empty( $value ) ) {
    return $clause;
}
```